src/main/java/seedu/address/ui/ExpandedPersonListPanel.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Person;
+
+/**
+ * Panel containing the view of an expanded person card.
+ */
+public class ExpandedPersonListPanel extends UiPart<Region> {
+    private static final String FXML = "ExpandedPersonListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(ExpandedPersonListPanel.class);
+
+    @FXML
+    private ListView<Person> personListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
+     */
+    public ExpandedPersonListPanel(ObservableList<Person> personList) {
+        super(FXML);
+        personListView.setItems(personList);
+        personListView.setCellFactory(listView -> new PersonListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
+     */
+    private class PersonListViewCell extends ListCell<Person> {
+        @Override
+        protected void updateItem(Person person, boolean empty) {
+            super.updateItem(person, empty);
+            if (empty || person == null) {
```

Comment:
Is it correct to understand this private class as a helper class that returns a `UiPart` in the constructor? If so, for the use case, should we even allow `null` graphics and text? 

i.e. should we have an assertion to check for nulls and disallow it?